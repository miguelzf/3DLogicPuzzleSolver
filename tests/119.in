- - - L Y
- - - - -
- - P G -
- - R - -
- - - - -

- - - - -
- - - - Y
- - - - -
- - - - -
- - - - -

G R - P -
- - - - -
- - L - -
- - - - -
- - - - -
