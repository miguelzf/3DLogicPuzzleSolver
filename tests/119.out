
Solution:
l  l  l  L  Y  
l  -  p  p  p  
l  -  P  G  y  
l  r  R  g  y  
l  r  -  g  y  
=============
l  r  -  g  y  ||  G  R  y  P  y  
l  r  -  g  Y  ||  g  r  y  y  y  
l  r  -  g  g  ||  g  r  L  l  l  
l  r  r  r  r  ||  r  r  -  -  l  
l  l  l  l  l  ||  l  l  l  l  l  
